### ISSUE TITLE
Elsa 2 - Designer weird behavior

### ISSUE THREAD

**Issue by riatdyl** [elsa-workflows/elsa-core#953]
Hello,

1. In the designer, I can hover over a branch without being on it with the cursor. 
if it is about 2 centimeters to the left the branch becomes red (selected). This is happening sometimes but I don't know why.

2. When I right click on a branch it splits the workflow in multiple branches. The problem is that we can't roll back and the workflow needs to be recreated.
Before right click :
![image](https://user-images.githubusercontent.com/43779018/117625979-e3245b00-b176-11eb-9d57-9ca93b234964.png)
After right click : 
![image](https://user-images.githubusercontent.com/43779018/117626045-fa634880-b176-11eb-8d10-2b9dcdb9ad22.png)

**Comment by sfmskywalker** [comment]
Interesting. We need to look into the hover thing. 

Right-clicking a connection will delete that connection, which causes the HTTP Response activity in your example to lose its parent, which means it becomes a root activity itself. This is expected behavior for now. We might change this in the future.

**Comment by tomy2105** [comment]
@riatdyl  you can, more or less "easily", recreate workflow by reconnecting back the connection that just got deleted.
Just SHIFT + click on plus of the Endpoint, release shift and click on Response and you'll have what you had when you started :).

**Comment by riatdyl** [comment]
Yes, I figured out it was possible to move the activities like this. :P Thank you ^^
